### PR TITLE
enable HOM autodetection by default (as in Crispy Doom)

### PR DESCRIPTION
**Pull request by rfomin**

### CODE REVIEW

File: Source/r_main.c [hunk 1]
```diff
@@ -620,7 +620,7 @@ void R_RenderPlayerView (player_t* player)
     { // killough 2/10/98: add flashing red HOM indicators
       byte c[47*47];
       extern int lastshottic;
-      int i , color = !flashing_hom || (gametic % 20) < 9 ? 0xb0 : 0;
+      int i , color = !flashing_hom || (gametic % 20) < 9 ? 0 : 0xb0;
```
Comment (fabiangreffrath) [inline Source/r_main.c:623]: Why did you change the swapping sequence?

**Review by fabiangreffrath — CHANGES_REQUESTED** [review]
Hm, okay, but why?

**Comment by rfomin** [comment]
> Hm, okay, but why?

HOM is annoying when the IDCLIP cheat is enabled, especially with automap in overlay mode.

File: Source/r_main.c (hunk 1 above)
Comment (rfomin) [reply Source/r_main.c:623]: Because by default I want to fill with black color, not red. Not sure I understand this code.

Comment (fabiangreffrath) [reply Source/r_main.c:623]: Ah, okay. So, `autodetect_hom` is the setting that can only be enabled by a cheat code whereas `flashing_hom` can be enabled by the menu.

Comment (fabiangreffrath) [reply Source/r_main.c:623]: Could you please add a comment why you chose to change the colors?

**Review by fabiangreffrath — CHANGES_REQUESTED** [review]

**Comment by rfomin** [comment]
After some thinking, I am no longer sure about this PR. This is one of those changes that no one notices and no one complains about. And I can always put together a version for personal use. Sorry to bother you.

**Comment by fabiangreffrath** [comment]
No problem, thank you.
